### PR TITLE
After Daniel's changes

### PR DESCRIPTION
**Pull request by dhrubpaul**
ai-core-aiapi-clientsdk-setup
- Step 1
- line 25, grammar
- line 27, sentence
- Step 2
- line 46, whitespace within bracket
- line 48, code highlight

- line 53, lowercase "install", line add
- line 65 <- ~line 85~.
- line 96, webage <- ~page~
- line 104, sentence
- line 108, cut-short
- line 128, italic fix
- line 215, code highlight.

ai-core-aiapi-clientsdk-repository
- Articles removed the Step headings, 1, 4, 7, 9
- Step 1:
- images re-ordered
- splitted into new steps (2 & 3)
- Step 2: added new
- Step 3: added new
- Step 4:
- code highlighted

ai-core-aiapi-clientsdk-resources
- Articles removed the step headings, 1, 4, 7, 9
- remove article from heading
- Step 1:
- description added
- code highlighted
- Steps heading in title case

ai-core-aiapi-clientsdk-workflows
- title and description fix
- Articles removed from step headings, 3, 5, 7
- Step 1:
- line 38: link corrected
- line 48: link corrected
- Step 2:
- code highlighted
- optional part removed
- Step 6:
- tabspace fix for code

ai-core-aiapi-clientsdk-train
- title fix for 2, 3, 5, 6, 7

ai-core-aiapi-clientsdk-deploy
- sentencecase for steps headings

ai-core-aiapi-postman-setup
- sentencecase and fix steps heading, 2, 5
- Step 1:
- table fix
- Step 3: newly added

ai-core-aiapi-postman-repository
- title fix
- sentencecase step heading 5, 6, 7, 8
- Step 2:
- fix description
- Step 4:
- code highlight
- Step 8:
- fix list
- Step 10:
- code highlight

ai-core-aiapi-postman-resource
-sentencecase step heading 1, 2, 3, 4
- Step 3:
- title fix
- Step 4:
- list fix image inline

ai-core-aiapi-postman-workflows
- titlefix
- article removed from step 3
- Step 2:
- code highlight
- Step 6:
- code highlight

ai-core-aiapi-postman-train
- sentencecase steps heading
- Step 5:
- header markdown fix
- Step 6:
- typo fix, COLLECTIONS <- ~COllECTIONS~
- sub-headin removed broken to Step 7
- table fix

ai-core-aiapi-postman-deploy
- sentencecase headings

### CODE REVIEW

**Comment by dhrubpaul** [comment]
@Losada, Helena, please promote

File: tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md [hunk 1]
```diff
@@ -110,15 +106,13 @@ Example output
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 4: ](Create a serving configuration and deployment)]
+[ACCORDION-BEGIN [Step 4: ](Create serving configuration)]
 
 
-Similar to the training we will now create a serving configuration. This time the input artifact is the trained model.
-We also create a deployment in the same step, since that is just one more line of code.
+Similar to the training we will now create a serving configuration, to bind the model with workflows. This time the input artifact is the trained model.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md:112]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md [hunk 2]
```diff
@@ -155,14 +149,12 @@ configuration for serving the model created
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 5: ](Create the serving deployment)]
+[ACCORDION-BEGIN [Step 5: ](Start serving deployment)]
 
-
-We create the serving deployment in a similar way that we did for the training execution.
+We will start the configuration for serving from previous step, this will be called **deployment**, its similar that we did for the training execution.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md:154]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md [hunk 3]
```diff
@@ -223,11 +215,9 @@ Deployment with d4bdc7b8df3f71d7 complete!
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 7: ](Using the deployed model)]
+[ACCORDION-BEGIN [Step 7: ](Using deployed ML model)]
 
-Inference the deployed model
-
-We will use the deployed model to perform inference. Ensure that the deployment is complete before we trigger the inference.
+We will use the deployed model to perform inference. Ensure that the deployment is complete before we make the inference.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md:220]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md [hunk 4]
```diff
@@ -313,11 +303,9 @@ Inference result: {'predictions': 'complaint'}
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 8: ](Stop the deployed model)]
-
-Stop the deployment
+[ACCORDION-BEGIN [Step 8: ](Stop deployed model)]
 
-**The Inference service runs 24/7. In order to reduce costs we stop it.**
+The Inference service runs 24/7. In order to reduce costs we will stop it.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-deploy/ai-core-aiapi-clientsdk-deploy.md:308]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md [hunk 5]
```diff
@@ -151,7 +154,7 @@ The content of the AI-Core service file should look something like this:
 }
 ```
 
-Store it inside the files folder: `files/aic_service_key.json`, we will use it to connect to the API in later steps, so ensure the path is correct.                    
+Store your SAP AI API service file inside the `files` folder: `files/aic_service_key.json`, we will use it to connect to the SAP AI Core in later steps, so ensure the path is correct.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-setup/ai-core-aiapi-clientsdk-setup.md:157]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-clientsdk-train/ai-core-aiapi-clientsdk-train.md [hunk 6]
```diff
@@ -120,13 +114,11 @@ Scenario ID 1 -> [text-clf-tutorial]
...
-[ACCORDION-BEGIN [Step 5: ](Register the training data as artifact)]
-
-An **Artifact** is reference to data or a file that is produced or consumed by an execution or deployment.
+[ACCORDION-BEGIN [Step 5: ](Register training data as artifact)]
 
-Previous we uploaded our dataset to AWS S3 and connected S3 to SAP AI Core.
+An **Artifact** is reference to data or a file that is produced or consumed by an execution *(training code)* or deployment.
 
-Now we have specifically point to that dataset of S3.
+Previous we uploaded our dataset to AWS S3 and connected S3 to SAP AI Core. Now we have specifically point to that dataset of S3.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-train/ai-core-aiapi-clientsdk-train.md:121]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-clientsdk-train/ai-core-aiapi-clientsdk-train.md [hunk 7]
```diff
@@ -219,19 +217,21 @@ Configuration created for running the training
 [ACCORDION-END]
 
 
-[ACCORDION-BEGIN [Step 7: ](Create the training execution)]
+[ACCORDION-BEGIN [Step 7: ](Create training execution)]
 
-After registering the artifact and creating a configuration for the execution, it is very simple to create an execution.
+To run configuration we start the execution.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-clientsdk-train/ai-core-aiapi-clientsdk-train.md:222]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-postman-deploy/ai-core-aiapi-postman-deploy.md [hunk 8]
```diff
@@ -207,19 +208,16 @@ Since it again takes some time for the our trained model to be deployed and read
...
 [ACCORDION-END]
 
-[ACCORDION-BEGIN [Step 4: ](Use Deployed Model)]
+[ACCORDION-BEGIN [Step 4: ](Use deployed model)]
 
-
-Inference the deployed model
-
-We will use the deployed model to perform inference. Ensure that the deployment is complete before we trigger the inference.
+We will use the deployed model to make inference. Ensure that the deployment is complete before we trigger the inference.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-postman-deploy/ai-core-aiapi-postman-deploy.md:213]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-postman-repository/ai-core-aiapi-postman-repository.md [hunk 9]
```diff
@@ -87,14 +92,14 @@ GitHub Desktop will only show **un-synced** changes.
...
 
 
-[ACCORDION-BEGIN [Step 4: ](API to Connect GitHub SAP AI Core)]
+[ACCORDION-BEGIN [Step 4: ](API to connect GitHub SAP AI Core)]
 
 
 > **CAUTION** At present SAP AI Core can connect to single GitHub repository, skip this step if its already connected to one. Contact support to change if required.
 
-After we connect GitHub repository to our SAP AI Core account, whenever we will push changes ( *upload files*) to GitHub, the SAP AI Core can pull them automatically.
+After we connect GitHub repository to our SAP AI Core account, whenever we will push changes *(upload files)* to GitHub, the SAP AI Core can pull(sync) them automatically.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-postman-repository/ai-core-aiapi-postman-repository.md:100]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

File: tutorials/ai-core-aiapi-postman-workflows/ai-core-aiapi-postman-workflows.md [hunk 10]
```diff
@@ -123,30 +123,43 @@ Visit <https://hub.docker.com>, go inside listed repository and view your upload
...
+Inside YAML files from previous step find the line
 
-If you want to have a more powerful computer hardware, for example with GPUs you can use a different resource plan.
+```YAML[2]
+  ...
+  ai.sap.com/resourcePlan: starter
+  ...
+```
+
+The `starter` is the computing resource plan we are using. Below is the list of other available plans offered by SAP AI Core.
```
Comment (MichaelCzcz) [inline tutorials/ai-core-aiapi-postman-workflows/ai-core-aiapi-postman-workflows.md:162]: Please don't say "we" unless you are referencing to SAP. Say "you" instead.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Thanks for making the changes. I have made some comments regarding style. Please change the parts I have commented on and also please check the rest of the tutorials for any occurrences of the "royal we" and change them accordingly.

**Pull request merged**